conf/test.config

```diff
@@ -25,4 +25,11 @@ params {
     // Genome references
     fasta               = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.fasta'
     fasta_fai           = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.fasta.fai'
+    gtf                 = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.gtf'
+
+    //STAR index
+    star_index          = 'https://github.com/nf-core/test-datasets/raw/modules/data/genomics/homo_sapiens/genome/index/star/star.tar.gz'
+
+    //STAR-Fusion
+    starfusion_genome   = '/hdd2/genome_builds/star_fusion/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir'
```

Comment:
this is your local path, there is no need for tests, not for this pipeline, to computationally demanding at the moment, event for aws

Reply:
@matq007 Thanks for the reviews. I did this for my internal testing. I shall remove it.

Reply:
I am wondering we can set the parameter to point to the actual compressed genome file in the web, like we use the star_index above. I will code the functionality with `UNTAR`, etc in the `prepare_genome.nf` and make it ready. When `--starfusion` is enabled in the command line it will be checked/executed, but will disable `(starfusion = false)` for now for testing.